spreadsheets/scalerank/landuse.csv

```diff
@@ -1 +1,72 @@
-kind,scalerank
+kind,operator,zoom::int,area::int,output:tier::int,output:scalerank::int
+national_park;park or protected land;battlefield,!United States Forest Service,>=3,>=300000000,1,3
```

Comment:
The difference between what's in the code and what you've got above is that the above also includes things which _do_ have `operator: "United States Forest Service"` and also have `protect_class` in that range. A quick check on the database shows that only 2% (79 / 3395) of USFS-operated features also have a protect class in that range. Features without the USFS operator property will already be included, regardless of their `protect_class`.

Given that `not: { operator: [ "United States Forest Service" ] }` is `true` for the majority of features, should that `any` be an `all`?